CameraOverlay/mqtt_message_test.py

```diff
@@ -0,0 +1,33 @@
+""" MQTT Test Script For Sending Messages To CameraOverlay """
+
```

Comment:
Now this is something more useful for this repository! (Although I would have made this into it's own Notion card). Anyway, can you please move this file into a `tests` folder. To move a file properly to maintain git history (within the `CameraOerlay` folder)
```
git mv mqtt_message_test.py tests/mqtt_message_test.py 
```